game/state/gamestate.cpp

```diff
@@ -654,31 +654,37 @@ void GameState::fillPlayerStartingProperty()
 	    auto v = current_city->placeVehicle(*this, {this, type}, this->getPlayer(), {this, bld});
 	    v->homeBuilding = v->currentBuilding;
 	}*/
+
 	for (auto &pair : this->initial_vehicles)
 	{
 		auto v = current_city->createVehicle(*this, pair.first, this->getPlayer(), {this, bld});
 		v->homeBuilding = v->currentBuilding;
 		for (auto &eq : pair.second)
 		{
-			v->addEquipment(*this, eq);
+			auto device = v->addEquipment(*this, eq);
+			device->ammo = eq->max_ammo;
 		}
 	}
+
 	// Give the player initial vehicle equipment
 	for (auto &pair : this->initial_vehicle_equipment)
 	{
 		base->inventoryVehicleEquipment[pair.first.id] = pair.second;
 	}
+
```

Comment:
These extra lines aren't really needed, and if you really think they make it clearer it should be a separate change anyway

Reply:
Sorry, which two lines? The two blank ones in the section above or all the changes to gamestate.cpp

If just the blank ones, then agreed, they are not needed im happy for them to be removed

Reply:
The blank ones - separating out logical changes into different commits (or even PRs) often makes things easier going forward - when doing stuff like merges or rebases like this, for example :)

So each "single change", be it bug fix, feature, or formatting change, ideally should each be in different changes.